### ISSUE TITLE
[Request Implemented 1.309] Unity shows trigger error for model

### ISSUE THREAD

**Issue by Hisashimaru** [LogicalError/realtime-CSG-for-unity#31]
If you enabled "Model Is Trigger" with CSG Model inspector, Unity outputs errors. No errors, if you enabled "Set Collider To Convex" before.

```
Triggers on concave MeshColliders are not supported
UnityEngine.Collider:set_isTrigger(Boolean)
InternalRealtimeCSG.MeshInstanceManager:Refresh(GeneratedMeshInstance, CSGModel) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/Managers/MeshInstanceManager.cs:369)
InternalRealtimeCSG.MeshInstanceManager:UpdateHelperSurfaceVisibility(HelperSurfaceFlags) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/Managers/MeshInstanceManager.cs:496)
RealtimeCSG.CSGSceneManager:Refresh(Boolean) (at Assets/Plugins/RealtimeCSG/Editor/Scripts/Managers/CSGSceneManager.cs:2374)
RealtimeCSG.UnitySceneEventHandler:RunEditorUpdate() (at Assets/Plugins/RealtimeCSG/Editor/Scripts/Managers/UnitySceneEventHandler.cs:265)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()
```

**Comment by LogicalError** [comment]
this has to do with how Unity handles colliders/triggers. I'll add a warning I guess

**Comment by LogicalError** [comment]
This issue is fixed in version 1.309 (by adding a warning) which is available on the update site 
( http://prenominal.nl/updates/index.php )
I will close this ticket when it's available on the asset store

**Comment by LogicalError** [comment]
Fix is in the version that's on the asset store now, so closing ticket
